src/mscorlib/shared/System/Text/StringBuilder.cs

```diff
@@ -1334,161 +1237,62 @@ public StringBuilder Insert(int index, char[] value, int startIndex, int charCou
-            return Insert(index, value.ToString(), 1);
-        }
+        public StringBuilder Insert(int index, float value) => Insert(index, value.ToString(), 1);
 
-        // Returns a reference to the StringBuilder with ; value inserted into 
-        // the buffer at index. Existing characters are shifted to make room for the new text.
-        // The capacity is adjusted as needed. If value equals String.Empty, the
-        // StringBuilder is not changed. 
-        // 
-        public StringBuilder Insert(int index, double value)
-        {
-            Contract.Ensures(Contract.Result<StringBuilder>() != null);
-            return Insert(index, value.ToString(), 1);
-        }
+        public StringBuilder Insert(int index, double value) => Insert(index, value.ToString(), 1);
 
-        public StringBuilder Insert(int index, decimal value)
-        {
-            Contract.Ensures(Contract.Result<StringBuilder>() != null);
-            return Insert(index, value.ToString(), 1);
-        }
+        public StringBuilder Insert(int index, decimal value) => Insert(index, value.ToString(), 1);
 
-        // Returns a reference to the StringBuilder with value inserted into 
-        // the buffer at index. Existing characters are shifted to make room for the new text.
-        // The capacity is adjusted as needed. 
-        // 
         [CLSCompliant(false)]
-        public StringBuilder Insert(int index, ushort value)
-        {
-            Contract.Ensures(Contract.Result<StringBuilder>() != null);
-            return Insert(index, value.ToString(), 1);
-        }
+        public StringBuilder Insert(int index, ushort value) => Insert(index, value.ToString(), 1);
 
-        // Returns a reference to the StringBuilder with value inserted into 
-        // the buffer at index. Existing characters are shifted to make room for the new text.
-        // The capacity is adjusted as needed. 
-        // 
         [CLSCompliant(false)]
-        public StringBuilder Insert(int index, uint value)
-        {
-            Contract.Ensures(Contract.Result<StringBuilder>() != null);
-            return Insert(index, value.ToString(), 1);
-        }
+        public StringBuilder Insert(int index, uint value) => Insert(index, value.ToString(), 1);
 
-        // Returns a reference to the StringBuilder with value inserted into 
-        // the buffer at index. Existing characters are shifted to make room for the new text.
-        // The capacity is adjusted as needed. 
-        // 
         [CLSCompliant(false)]
-        public StringBuilder Insert(int index, ulong value)
-        {
-            Contract.Ensures(Contract.Result<StringBuilder>() != null);
-            return Insert(index, value.ToString(), 1);
-        }
+        public StringBuilder Insert(int index, ulong value) => Insert(index, value.ToString(), 1);
 
-        // Returns a reference to this string builder with value inserted into 
```

Comment:
@AlexGhiondea I believe this is fairly obvious from the naming of the method. The comment only takes up a lot of space.